drivers/clickhouse/clickhouse_test.go

```diff
@@ -116,10 +116,10 @@ func TestColumns(t *testing.T) {
 	if err != nil {
 		log.Fatalf("could not read columns: %v", err)
 	}
-	checkNames(t, "column", res, colNames()...)
+	checkNames(t, "column", true, res, colNames()...)
 }
 
-func checkNames(t *testing.T, typ string, res interface{ Next() bool }, exp ...string) {
+func checkNames(t *testing.T, typ string, checkForAdditional bool, res interface{ Next() bool }, exp ...string) {
```

Comment:
I'm not sure `checkForAdditional` is necessary, can we always check?

Reply:
I was thinking about the case where new functions gets added in the newer versions of clickhouse.

Reply:
Then we should pin the version of CH we use for testing to a particular one and fix tests whenever we wtorek update it.

Reply:
Yeah. That is a good idea. I will make the necessary changes.